.github/workflows/test-deploy.yml

```diff
@@ -0,0 +1,69 @@
+name: Test superset deployment
+
+on:
+  pull_request:
+    types:
+      - opened
+    branches:
+      - main
+
+jobs:
+  test_local:
```

Comment:
What do you think about running test local on all prs? How long does it take to run?

Reply:
It could take somewhere between 3-5 minutes and I don't see why we couldn't. To be clear, would the trigger then be a push to any branch OR a PR opened to main and/or rc?

Reply:
The trigger for test local would be prs to rc/main. The same rules as for running out other tests. Not just when the pr opens.